terraform/modules/kubernetes/aks_redis.tf

```diff
@@ -25,7 +25,7 @@ resource "kubernetes_deployment" "redis" {
         }
         container {
           name  = local.redis_service_name
-          image = "redis:6-alpine"
+          image = "redis:5-alpine"
```

Comment:
Can we set it to [6.0-alpine](https://hub.docker.com/layers/library/redis/6.0-alpine/images/sha256-f8811e0e863b456a821b456976d1e8601b60f7d0833e1811c0f54d010c740cd3?context=explore) to reflect what it is used in production?

Reply:
Done!